articles/partner-twilio-php-how-to-use-voice-sms.md

```diff
@@ -141,33 +151,38 @@ As mentioned, this code uses a Twilio-provided site to return the TwiML response
 ## <a id="howto_send_sms"></a>How to: Send an SMS message
 The following shows how to send an SMS message using the **Services_Twilio** class. The **From** number is provided by Twilio for trial accounts to send SMS messages. The **To** number must be verified for your Twilio account prior to running the code.
 
-    // Include the Twilio PHP library.
-    require_once 'Services/Twilio.php';
-
-    // Library version.
-    $version = "2010-04-01";
-
-    // Set your account ID and authentication token.
-    $sid = "your_twilio_account_sid";
-    $token = "your_twilio_authentication_token";
-
-
-    $from_number = "NNNNNNNNNNN"; // With trial account, texts can only be sent from your Twilio number.
-    $to_number = "NNNNNNNNNNN";
-    $message = "Hello world.";
-
-    // Create the call client.
-    $client = new Services_Twilio($sid, $token, $version);
-
-    // Send the SMS message.
-    try
-    {
-        $client->$client->account->messages->sendMessage($from_number, $to_number, $message);
-    }
-    catch (Exception $e) 
-    {
-        echo 'Error: ' . $e->getMessage();
-    }
+```php
+// Required if your environment does not handle autoloading
+require __DIR__ . '/vendor/autoload.php';
+
+// Use the REST API Client to make requests to the Twilio REST API
+use Twilio\Rest\Client;
+
+// Your Account SID and Auth Token from twilio.com/console
+$sid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX';
+$token = 'your_auth_token';
+$client = new Client($sid, $token);
+
+$from_number = "NNNNNNNNNNN"; // With trial account, texts can only be sent from your Twilio number.
+$to_number = "NNNNNNNNNNN";
+$message = "Hello world.";
```

Comment:
Do you think it's a good idea to update the "prose" on this line from "Hello world." to "Hello World!"?